src/NUnitTestAdapter/TestLogger.cs

```diff
@@ -48,7 +48,7 @@ public void LoadingAssemblyFailedWarning(string dependentAssembly, string source
 
         public void NUnitLoadError(string sourceAssembly)
         {
-            SendErrorMessage("NUnit failed to load " + sourceAssembly);
+            SendInformationalMessage("NUnit failed to load " + sourceAssembly);
```

Comment:
A NonRunnable assembly seem like at least a potential error. For example, if I rename a text file to ".dll" it will be a non-runnable assembly. It sounds like we need some further distinction based on why it's not runnable. Issue nunit/nunit#1026 is supposed to address this but in the meantime, we had planned to use the message to distinguish.